src/Client/Collections.php

```diff
@@ -53,6 +54,19 @@ public function forSubject(string ...$subjectId) : self
         return $clone;
     }
 
+    public function containing(string ...$items) : self
```

Comment:
Should this replace `getId()`? Means you'd have to call something like `getId()->getId()` to get the ID. Alternatively there'd be two methods (eg `getId()` and `getIdentifier()`), which seems a bit odd.